app.js

```diff
@@ -41,12 +44,26 @@ app.use(cors());
 app.use(logger('dev'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
-app.use(cookieParser());
+app.use(cookieParser()); // DO NOT MODIFY THIS LINE, IT MAY SCREW UP EXPRESS-SESSION
```

Comment:
This comment applies to cases when the cookieParser gets a secret. in our case it does not.
as a matter of fact I left cookieParser just in case you use it for something. I can remove it completely